### DISCUSSION TITLE
NOT SEEING MY FILES IN MY REPOSITORY

### DISCUSSION THREAD

**Discussion by Godwin987** [community/community#42990] [Repositories] [answered]
### Select Topic Area

Question

### Body

I pushed my files to my repository using git but i am not seeing them there even after  refreshing.

**Comment by LuisContrerasGlz** [comment]
There could be a couple of things that are making this issue, check if you are pushing to the correct branch and repository
Also check the output of the push command to see if there were any errors, or any file conflicts.
You can also to try creating a new commit and pushing to the repository

**Reply by Godwin987** [reply]
There were no errors

**Comment by jfullstackdev** [comment]
these are your latest repos 

![image](https://user-images.githubusercontent.com/47092464/210360137-e1a2c5c7-0948-4b26-aca2-4f8eaae3a5ff.png)

are those the repos you are referring to ?

**Reply by Godwin987** [reply]
![Screenshot_20230103-145344](https://user-images.githubusercontent.com/104033792/210370969-7f3fda94-b148-4987-bde0-f1a38446e963.png)

This one. Scroll down you'll see it

**Comment by Ankush11903** [comment]
Because U have deleted the Files from this repo.
on first 2 commits u have added those files but after that u have made 3 commits in which all files are deleted.
![Screenshot (627)](https://user-images.githubusercontent.com/91266368/210375416-1745e85d-4844-4e09-b81e-4acf2e69bd3e.png)

**Comment by Ankush11903** [comment, marked as answer]
![b](https://user-images.githubusercontent.com/91266368/210375573-18236734-400f-4b47-b577-f639e23f104d.png)
![a](https://user-images.githubusercontent.com/91266368/210375581-36247ca6-7239-45fd-a684-c4c9c1edb4e7.png)
see.
please marked it as answered if it helps you. :)
